testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/nuget/NugetITSupport.java

```diff
@@ -76,7 +77,9 @@
   public static Option[] configureNexus() {
     return options(NexusHttpsITSupport.configureNexus(),
         wrappedBundle(maven("org.apache.httpcomponents", "httpmime").versionAsInProject()),
-        mavenBundle("org.sonatype.http-testing-harness", "server-provider").versionAsInProject()
+        mavenBundle("org.sonatype.http-testing-harness", "server-provider").versionAsInProject(),
+        wrappedBundle(maven("org.sonatype.nexus", "nexus-test-concurrent").versionAsInProject())
+            .overwriteManifest(OverwriteMode.FULL).instructions("DynamicImport-Package=*")
```

Comment:
FYI you only need `overwriteManifest(OverwriteMode.FULL)` when you want to replace existing OSGi metadata; since this is a plain jar it's not necessary here. Also I didn't see any classloading in `nexus-test-concurrent` so is the dynamic import actually required?

Since `nexus-test-concurrent` is under our control why not change the pom to use `<packaging>bundle</packaging>`? Then you could use `mavenBundle` and avoid the cost of wrapping it at runtime. See `nexus-siestajs-testsupport` for example.